frontend/src/store/activeProfile.ts

```diff
@@ -0,0 +1,53 @@
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import { setActiveProfileAuto } from "./helpers/profileHelper";
+import { setActiveProfile } from "./helpers/profileHelper";
+import { ProfileInterface } from "../interfaces";
+
+const initialState = {
+    status: "",
+    activeProfile: {} as ProfileInterface,
```

Comment:
issue: the as keyword is used incorrectly here. Usually it's used for casting a variable to a specific type. If you're meaning to set the type for this object property, you should define an interface instead.

```
interface ProfileState = {status: string; activeProfile: ProfileInterface}
const initialState: ProfileState = {};
```